firstvoices/firstvoices/settings.py

```diff
@@ -364,3 +349,63 @@
 else:
     EMAIL_SENDER_ADDRESS = os.getenv("EMAIL_SENDER_ADDRESS", "sender@example.com")
     EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+
+
+# LOGGING SETUP
+ELASTICSEARCH_LOGGER = "elasticsearch"
+
+# Filtering logs to console
+console_filters = []
+log_handlers = ["console"]
+
+if not DEBUG:
+    console_filters += ["debug_info_log_level_filter"]
+    log_handlers += ["file"]
+
+# Based on the default settings
+# ref: https://docs.djangoproject.com/en/5.1/ref/logging/#default-logging-configuration
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "filters": {
+        "debug_info_log_level_filter": {
+            "()": "firstvoices.logging_utils.DebugInfoLogLevelFilter"
+        },
+    },
+    "formatters": {
+        "standard": {
+            "format": "{levelname}-{asctime}:{pathname} -- {message}",
+            "style": "{",
+        },
+        "detailed": {
+            "format": "{levelname}-{asctime}:{pathname} {process:d} {thread:d} -- {message}",
+            "style": "{",
+        },
+    },
+    "handlers": {
+        "console": {
+            "level": "DEBUG",  # DEBUG + INFO
+            "filters": console_filters,
+            "class": "logging.StreamHandler",
+            "formatter": "standard",
+        },
+        "file": {
+            "level": "WARNING",  # WARNING + ERROR
+            "class": "logging.FileHandler",
+            "filename": "/var/log/django/app.log",
```

Comment:
That's right. We might need to create that directory in our Dockerfile too. 

RUN mkdir -p /var/log/django